ydb/core/base/tablet_pipecache.h

```diff
@@ -74,14 +74,24 @@ struct TEvPipeCache {
 
     struct TEvDeliveryProblem : public TEventLocal<TEvDeliveryProblem, EvDeliveryProblem> {
         const ui64 TabletId;
+        const NKikimrProto::EReplyStatus ConnectStatus;
```

Comment:
More grammatically correct: ConnectionStatus

Reply:
Well, it's the status of an initial connect, not of any connection.

Reply:
But I'm going to replace it with `bool Connected` anyway.